### DISCUSSION TITLE
Workflow Automation - parallel flows, loops, monitoring

### DISCUSSION THREAD

**Discussion by ranjith-murugasamy** [Budibase/budibase#5878] [Ideas]
It would be great if the workflow automation has the following
1. parallel flows based on conditional checks
2. Lopping back to any of the previous steps based on condition
3. monitoring screen to see the transactions and their status

**Comment by melohagan** [comment]
For number one, do you mean conditional branching e.g. If/Else?

The looping back to a 'goto' would be cool, although it could be tricky to incorporate conditions into that. 

Number 3 is coming soon (maybe the next release or two), among other app level stats :)
